pkg/crc/machine/machine.go

```diff
@@ -255,17 +255,17 @@ func Start(startConfig StartConfig) (StartResult, error) {
 	}
 
 	// Check the certs validity inside the vm
+	needsCertsRenewal := false
 	logging.Info("Verifying validity of the cluster certificates ...")
 	expiringIn7Days, duration, err := cluster.CheckCertsValidity(sshRunner)
 	if err != nil {
-		result.Error = err.Error()
-		return *result, errors.New(err.Error())
-	}
-	// Only show when VM is started from stopped state.
-	if exists {
-		if expiringIn7Days {
+		needsCertsRenewal = true
+	} else if expiringIn7Days {
+		// Only show when VM is started from stopped state.
+		if exists {
 			logging.Warnf("Bundle certificates are going to expire in %d days, better to use new release", duration)
 		}
+		needsCertsRenewal = true
```

Comment:
@cfergeau why we need to cert renewal if there is already 7 days remaining?

Reply:
Initial idea was to be proactive, and to renew the certs before they even expire. But since the renewal process is quite long, only doing it when absolutely required is a good idea. I'll change it.

Reply:
@cfergeau sure, please change it , it is really add around 3mins to complete process.